### PR TITLE
Update Conversation naming

### PR DESCRIPTION
**Pull request by stevenpkg**

### CODE REVIEW

**Comment by stevenpkg** [comment]
@germanattanasio please do not approve till the 15th

**Comment by ttlew** [comment]
@stevenpkg can you add `(formerly Conversation)` after the new name? not after every mention, but the initial mentions and titles/subtitles

**Comment by stevenpkg** [comment]
@ttlew Okay

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Environment variables
 WORKSPACE_ID=
-CONVERSATION_USERNAME=
-CONVERSATION_PASSWORD=
+WATSON_ASSISTANT_USERNAME=
```
Comment (germanattanasio) [inline .env.example]:
```
ASSISTANT_USERNAME
ASSISTANT_PASSWORD
```

File: manifest.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 declared-services:
-  my-conversation-service:
```
Comment (germanattanasio) [inline manifest.yml:3]: if you use `watson-assistant-service` above then you need to update this to be the same

File: routes/conversation.js [hunk 3]
```diff
@@ -18,10 +18,10 @@ const watson = require('watson-developer-cloud'); // watson sdk
 
 // Create the service wrapper
 const conversation = new watson.ConversationV1({
-  // If unspecified here, the CONVERSATION_USERNAME and CONVERSATION_PASSWORD env properties will be checked
+  // If unspecified here, the WATSON_ASSISTANT_USERNAME and WATSON_ASSISTANT_PASSWORD env properties will be checked
```
Comment (germanattanasio) [inline routes/conversation.js]: `ASSISTANT_USERNAME`, `ASSISTANT_PASSWORD`

**Review by germanattanasio — CHANGES_REQUESTED** [review]

File: manifest.yml (hunk 2 above)
Comment (stevenpkg) [reply manifest.yml:3]: yes had fixed this

File: routes/conversation.js (hunk 3 above)
Comment (stevenpkg) [reply routes/conversation.js]: done

**Review by germanattanasio — APPROVED** [review]

**Pull request merged**